tests/client-dbus/tests/udev/_loopback.py

```diff
@@ -134,10 +139,11 @@ def device_files(self, tokens):
         :return: The list devices corresponding to the tokens
         :rtype: list of str
         :raises: KeyError if any token is not found
-        :raises: AssertionError if any devnode is None
+        :raises: RuntimeError if any devnode is None
         """
         result = [self.devices[token][0] for token in tokens]
-        assert all([devnode is not None for devnode in result])
+        if not all([devnode is not None for devnode in result]):
```

Comment:
Might as well drop the double negative here, while you're doing the other changes:

```
if any([devnode is None for devnode in result]):
```